webdriver-spec.html

```diff
@@ -3342,21 +3342,47 @@
  Where a <a>command</a> is not supported,
  an <a>unsupported operation</a> <a>error</a> is returned.
 
-<p>A <a>top-level browsing context</a>’s <dfn>window size</dfn>
- are the outer dimensions, including any browser chrome
- externally drawn window decorations in <a>CSS reference pixels</a>
- of the operating system window corresponding to it.
- Its <a>JSON representation</a> is the following:
+<p>A <a>top-level browsing context</a>’s <dfn>window size</dfn> are
+ the outer dimensions, including any browser chrome externally drawn
+ window decorations in <a>CSS reference pixels</a> of the operating
+ system window corresponding to it.  The <dfn>JSON serialization of
+ window size</dfn> is a JSON <a>Object</a> initialized with:
+
+ <dl>
+  <dt>width
+  <dd>Width of the <a>top-level browsing context</a>’s outer
+   dimensions, including any browser chrome and externally drawn window
+   decorations in <a>CSS reference pixels</a>.
+
+  <dt>height
+  <dd>Height of the <a>top-level browsing context</a>’s outer
+   dimensions, including any browser chrome and externally drawn window
+   decorations in <a>CSS reference pixels</a>.
+ </dl>
+
 
 <p class=note>In some user agents the operating system’s
  window dimensions including decorations,
  are provided by the proprietary <code>window.outerWidth</code>
  and <code>window.outerHeight</code> [[!DOM]] properties.
 
-<p>A <a>top-level browsing context</a>’s <dfn>window position</dfn>
- is defined as a dictionary of the <a>screenX</a> and <a>screenY</a> attributes
- on the <a><code>WindowProxy</code></a> it corresponds to.
- Its <a>JSON representation</a> is the following:
+<p>A <a>top-level browsing context</a>’s <dfn>window position</dfn> is
+ defined as a dictionary of the <a>screenX</a> and <a>screenY</a>
+ attributes on the <a><code>WindowProxy</code></a> it corresponds to.
+ The <dfn>JSON serialization of window position</dfn> is a
```

Comment:
We don’t _need_ a separate definition for each JSON serialisation if we think of it as a trait, but I guess this OK to make the link point to the right location.